lib/jekyll/converters/markdown/kramdown_parser.rb

```diff
@@ -100,8 +105,10 @@ def strip_coderay_prefix(hash)
         private
         def modernize_coderay_config
           if highlighter == "coderay"
-            Jekyll::Deprecator.deprecation_message "You are using 'kramdown.coderay' in your configuration, " \
+            Jekyll::Deprecator.deprecation_message(
+              "You are using 'kramdown.coderay' in your configuration, " \
               "please use 'syntax_highlighter_opts' instead."
```

Comment:
Yeah, could we just say "kramdown.coderay is deprecated. Use kramdown.syntax_highlighter_opts instead.", or is that still really long?